cli/kata-check_ppc64le.go

```diff
@@ -55,6 +63,13 @@ func hostIsVMContainerCapable(details vmContainerCapableDetails) error {
 		return err
 	}
 
+	out, err := exec.Command("bash", "-c", checkPower8cmd).Output()
```

Comment:
Please can we not do this. The code already parses `/proc/cpuinfo` - see `getCPUInfo()`.

Reply:
@jodh-intel : Addressed the issue.